### ISSUE TITLE
ISSUE: first pixel is never shown

### ISSUE THREAD

**Issue by casainho** [jposada202020/CircuitPython_uplot#5]
I did a test, to show a list with only 1 element and I never see it on the display. But if I increase to 2 elements, then I can see and it seems to be the 2 elements - see the small 2 pixels horizontal line:

![image](https://user-images.githubusercontent.com/147128/224018984-959cd6db-8a9c-4d56-a656-f7ad21f1c213.png)

**Comment by jposada202020** [comment]
Yes, that is correct. The library uses a Polygon two trace the line, so it will need two points to do it.  As the logic for the logging data is different from the Cartesian plane, move it to a different submodule, this will make it easier to treat corner cases like this. Will work over the weekend on that :)

**Comment by jposada202020** [comment]
Please see 0.9.0. thanks :) and the example https://github.com/jposada202020/CircuitPython_uplot/blob/main/examples/uplot_ulogging.py
